modules/internal/ChapelError.chpl

```diff
@@ -53,6 +53,14 @@ module ChapelError {
     }
   }
 
+  class NilThrownError : Error {
+    const nil_thrown_msg = "thrown error was nil";
```

Comment:
I don't really see why not just return this string in message(), but it's not a big deal either way.